### ISSUE TITLE
Fields - Subform > Fields - Media : Add Multiple

### ISSUE THREAD

**Issue by pnkr** [joomla/joomla-cms#37847]
### Steps to reproduce the issue
Adding a custom field Media type and then select it within a Subform field.
Then in an article you can add multiple media for the subform. But if you have a lot of images you must do that one by one for each image which is exausting.

### Expected result
Multiple select the images you want and add them to the subform

### Actual result
Now you just need to add one by one each image. This is really unproductive.

### System information (as much as possible)
Joomla 4.1.3

**Comment by drmenzelit** [comment]
I think the media field is not the correct one for what do you want to do, I would suggest to use the `imagelist` field that allows multiple select.

**Comment by pnkr** [comment]
Imagelist is even worst. It checks a specific predefined folder and you cannot see what images you are selecting!
But even if you could that means that you should first upload images in that bucket folder and then go there and select them.

Both ways work. I am just pointing the problem from a UX point of view. Its not productive. If by some way we could let the user select the folder in imagelist field at least it would be ok

**Comment by JackKellyUK** [comment]
I would argue this is negative for accessibility. Widgetkit has this feature and when the accessibility audit came around for a client none of the images had alt tags as expected - wasn't fun converting them all.

**Comment by pnkr** [comment]
> I would argue this is negative for accessibility. Widgetkit has this feature and when the accessibility audit came around for a client none of the images had alt tags as expected - wasn't fun converting them all.

Media field adds alt field with the image so it’s not an accessibility issue. If the template does not handle this well it’s a template problem.

**Comment by chmst** [comment]
@JackKellyUK thanks for mentioning a11y. 
But in this case, as @pnkr says, it could possible to add an alt-text to every image. So I think it is a good feature request.
